packages/web-components/fast-foundation/src/anchor/anchor.template.ts

```diff
@@ -38,10 +38,11 @@ export const AnchorTemplate = html<Anchor>`
         aria-owns="${x => x.ariaOwns}"
         aria-relevant="${x => x.ariaRelevant}"
         aria-roledescription="${x => x.ariaRoledescription}"
+        ${ref("root")}
```

Comment:
Kinda a nit, but can we call this "control" instead of "root" so that aligns with the class and part attributes?

Reply:
This is my bad - probably need a change to button as well :|

Reply:
Will change.